app/serializers/search_result_serializer.rb

```diff
@@ -1,10 +1,6 @@
 class SearchResultSerializer < ActiveModel::Serializer
   embed :id, include: true
-  attributes :id, :priority, :query, :discussion_blurb, :motion_blurb, :comment_blurb
-
-  has_one :discussion
-  has_one :motion, root: :proposals
```

Comment:
This serialization was killing us as well, especially given that the discussion serializer had some perf issues.

Now we just get a slightly-dumbed-down discussion back as the result, with no bells or whistles.